src/modules/0.0.17/aws_acm_import/index.ts

```diff
@@ -29,6 +29,19 @@ class CertificateImportMapper extends MapperBase<CertificateImport> {
     return arn;
   }
 
+  db = new Crud2<CertificateImport>({
+    create: (es: CertificateImport[], ctx: Context) => ctx.orm.save(CertificateImport, es),
+    update: (es: CertificateImport[], ctx: Context) => ctx.orm.save(CertificateImport, es),
+    delete: (es: CertificateImport[], ctx: Context) => ctx.orm.remove(CertificateImport, es),
+    read: async (ctx: Context, id?: string) => {
+      const opts = id ? {
+        where: {
+          id,
+        }
+      } : {};
+      return await ctx.orm.find(CertificateImport, opts);
+    },
+  });
```

Comment:
Due to a bug in how we were detecting `@cloudId` vs `entityId` usage, these were being autogenerated *incorrectly* before, and only worked for our usage by chance.

Fixed that bug and now added these in.